app/meldingen/admin/containers/IncidentDetailPage/index.js

```diff
@@ -0,0 +1,83 @@
+/**
+ *
+ * IncidentDetailPage
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { Link } from 'react-router-dom';
+
+import injectSaga from 'utils/injectSaga';
+import injectReducer from 'utils/injectReducer';
+import { makeSelectLoading, makeSelectError } from 'containers/App/selectors';
+import makeSelectIncidentDetailPage from './selectors';
+import reducer from './reducer';
+import saga from './saga';
+import messages from './messages';
+import './style.scss';
+
+import { requestIncident } from './actions';
+
+
+export class IncidentDetailPage extends React.Component { // eslint-disable-line react/prefer-stateless-function
```

Comment:
`// eslint-disable-line react/prefer-stateless-function`
Do we have to have this line for every component?